extras/userdoc/md/documentation/installation.md

```diff
@@ -235,145 +218,211 @@ support.
 
         xcode-select --install
 
-3.  Install the GNU Compiler Collection, the GNU Science Library, and Python
-    from [MacPorts](http://en.wikipedia.org/wiki/Macports) (you could use Fink
-    or Homebrew, too, but instructions here are based on MacPorts; if you
-    have installed a complete Python from another source, e.g. Anaconda,
-    you do not need Python from MacPorts):
-
-        sudo port install gcc48                   # GCC compiler collection
-        sudo port select gcc mp-gcc48      # make gcc-48 the default compiler
-        sudo port install gsl +gcc48           # install GNU Science Library,
-        use GCC 4.8 compiler
-        sudo port install autoconf automake libtool    # build tools
-
-        # now the Python stuff
-        sudo port install python27
-        sudo port select python python27   # make MacPorts Python 2.7 the default
-        sudo port install py27-cython
-        sudo port select cython cython27   # make Python 2.7 python the default cython
-        sudo port install py27-numpy py27-scipy py27-matplotlib py27-ipython
-        sudo port select ipython ipython27   # set default
+3.  Install dependencies via Homebrew
+
+     brew install gcc cmake gsl open-mpi libtool
 
 4.  Create a directory for building and installing NEST (you should always build
     NEST outside the source code directory; installing NEST in a "place of its
     own" makes it easy to remove NEST later).
 
-5.  Extract the NEST tarball as a subdirectory in that directory, create a
-    build-directory next to the source code directory
+5.  Extract the NEST tarball as a subdirectory in that directory or clone NEST from Github into a subdirectory. 
 
-        cd ~                  # move to home directory
         mkdir NEST       # directory for all NEST stuff
         cd NEST
-        mkdir nest22    # directory for all NEST 2.2 stuff
-        cd nest22
-        tar zxf  nest-2.2.2.tar.gz
+        tar zxf nest-2.12.0.tar.gz
         mkdir bld
         cd bld
 
-6.  Configure and build NEST inside that directory:
+6.  Configure and build NEST inside the build directory:
+
+     cmake -DCMAKE_INSTALL_PREFIX:PATH=</install/path> \
+           -DCMAKE_C_COMPILER=gcc-6 \
+           -DCMAKE_CXX_COMPILER=g++-6 \
+           </path/to/NEST/src>
+
+     make -j4         # -j4 builds in parallel using 4 processes
+     make install
+     make installcheck
+
```

Comment:
Should be indented more to become code block. Alternatively, use triple back ticks throughout the document to denote code blocks.